packages/vm/tests/api/runTx.spec.ts

```diff
@@ -180,8 +180,9 @@ tape('runTx() -> API parameter usage/data errors', (t) => {
 
     try {
       await vm.runTx({ tx })
+      // TODO uncomment:
+      // t.fail('should throw error')
```

Comment:
TODO

Reply:
Was also irritated here at first but now had a closer look and ran locally. On second look it seems that this was just not working before and Ryan has just added an additional comment with a TODO that this `t.fail()` call should not be reached (which it is). This was not working before the PR though as well and the issue is unrelated, so it's just a net improvement here and not some left-over comment.

I guess this tests generally has not yet found its place respectively I remember that I also already stumbled upon it. We just not have this coherently in place how to deal with these kind of situations with two different `Common` instances. 

We likely want to first give these checks from here https://github.com/ethereumjs/ethereumjs-monorepo/issues/969 on Common consistency some more thought before we give this test more attention. This is rather just a test atm without the necessary implementation. 😋

P.S.: have removed the other TODO, this was actually not necessary (any more?).

Reply:
I just checked but this `t.fail` should be called, right? The VM is on istanbul and the tx is on Berlin, so running access list txs on Istanbul should be invalid. Now, the test passes if the catch clause is not reached.

Reply:
Yes - again - therefore the "TODO" from Ryan here. Please read my longer comment from above (if you haven't already) with a suggestion on how to proceed here.